packages/suite/src/support/messages.ts

```diff
@@ -4754,8 +4754,7 @@ export default defineMessages({
     },
     TR_CHARGE_TO_50_ATLEAST: {
         id: 'TR_CHARGE_TO_50_ATLEAST',
-        defaultMessage:
-            'Charge your Trezor to at least 50% or power it with a cable to prevent interruptions.',
+        defaultMessage: 'Charge your Trezor to at least 40% to prevent interruptions.',
```

Comment:
On mobile, we pass in the threshold constant. Can it be done here as well?

Reply:
Yes, we should pass a value.

Reply:
Fixed here, however I had to change the string key. Not sure whether this will break something on crowdin maybe? @komret 
https://github.com/trezor/trezor-suite/pull/22116/commits/112b2866f38d78446e6e53e0aade08c40c6343e0